packages/patterns/src/patterns/patternMatchers.js

```diff
@@ -1788,25 +1792,52 @@ const makeAwaitArgGuard = argPattern => {
   return result;
 };
 
-const PatternListShape = M.arrayOf(M.pattern());
+// M.rawValue()
+
+// TODO does not need to be a singleton, and would not be if we added a
+// parameter, like a description string.
+/** @type {RawValueGuard} */
+const TheRawValueGuard = harden({
+  klass: 'rawValueGuard',
```

Comment:
We have completely moved away from `klass: '...',`. Please follow the new coding pattern. The hazard with the `klass: '...'` coding pattern is that if it happens to appear in another position where a pattern is expected, it will be interpreted as a key pattern that matches only itself. This unsoundness was actually the unsoundness flaw that pushed us away from `klass:`.